Tests/SentryTests/Helper/SentryFileManagerTests.swift

```diff
@@ -578,6 +578,29 @@ class SentryFileManagerTests: XCTestCase {
         XCTAssertNotNil(sut.readTimezoneOffset())
     }
 
+    func testReadPreviousBreadcrumbs() {
```

Comment:
`m`: Is this test covering all the new functionality added in SentryFileManager?

Reply:
I think so 😅 Depend on if you count this as testing `moveBreadcrumbsToPreviousBreadcrumbs` as well, which I think it does, but from your comment above it seems you think it doesn't?

Reply:
There is now also a SentrySDK test that tests `moveBreadcrumbsToPreviousBreadcrumbs`, so I think all functionality in SentryFileManager is tested 👍